Work in progress: anniversary blog post

bep opened the pull request.

bep commented:
@digitalcraftsman I created a PR for this to get the preview. Please add to it (numbers? There are some other stats floating around that makes us look good). 

I created an outline, will add "my story" tomorrow.

@kaushalmodi @spf13 @regisphilibert and gang ... input/suggestions.

kaushalmodi commented on content/en/news/lets-celebrate-hugos-5th-birthday/index.md:
```diff
@@ -0,0 +1,29 @@
+---
+date: 2018-07-04
+title: "Let’s celebrate Hugo’s 5th birthday"
```
Do you want to use a straight-quote here: "Hugo’s" -> "Hugo's"?

kaushalmodi commented on content/en/news/lets-celebrate-hugos-5th-birthday/index.md:
```diff
@@ -0,0 +1,29 @@
+
+Some notable sites.
+
+https://www.smashingmagazine.com/ https://vote.gov/ https://www.netlify.com https://gohugo.io/showcase/letsencrypt/ https://kubernetes.io/
```
Should these sites be put as individual list items?

bep replied:
I just dumped them in there... bokmarks for later. When I asked for input I meant on a more general level. This is a rough draft that I will finish tomorrow.

kaushalmodi commented on content/en/news/lets-celebrate-hugos-5th-birthday/index.md:
```diff
@@ -0,0 +1,29 @@
+
+* 26800+ [stars](https://github.com/gohugoio/hugo/stargazers) on GitHub. This makes us currently the second most starred static site generator, according to [StaticGen.com](https://www.staticgen.com/)[^sgen]
+* 444+ [contributors](https://github.com/gohugoio/hugo/graphs/contributors)
+* 235+ [themes](http://themes.gohugo.io/)
```
http -> https

moorereason commented on content/en/news/lets-celebrate-hugos-5th-birthday/index.md:
```diff
@@ -0,0 +1,29 @@
+---
+date: 2018-07-04
+title: "Let’s celebrate Hugo’s 5th birthday"
+description: "How a side project became the fastest and one of the most popular framework for building websites."
```
Strike "the fastest and" and make "framework" plural.

```yaml
description: "How a side project became one of the most popular frameworks for building websites."
```

digitalcraftsman commented:
I'll try to create a line chart that shows the rise of Hugo from release to release using the GitHub stars as metric.

Regarding content suggestions: static site generators became in vogue over last couple of years. Hugo was already there and took off. Just look at companies like Forestry and Netlify that emerged and build an infrastructure and tools around Hugo / Jekyll.

I'm not sure what @bep planned for tomorrow but it might be interesting to know why our lead developer jumped on this train. Why Go? Why an SSG like Hugo?

Furthermore, it would be interesting for a lot of people what @bep's vision is for the future of Hugo. What needs to be done to get v1.0 released?

Btw, I like the addition of the Hugo gopher to the artwork :smile:

digitalcraftsman commented:
Here are two diagrams to illustrate Hugo's development in terms of GitHub stars and the number of themes:

![themes and releases](https://user-images.githubusercontent.com/7010165/42247726-55cd714e-7f22-11e8-8b40-ea88330c4190.png)
![github stars and releases](https://user-images.githubusercontent.com/7010165/42247727-55f6f118-7f22-11e8-9cf3-286eb66c59c5.png)

bep commented:
@digitalcraftsman these are great, but I suspect that a time based graph would paint a more accurate picture. We at some point went from a 6 month to a 5 week release cycle, which obviously makes a different scale.

digitalcraftsman commented:
I replaced the version numbers with the dates of the releases. Google spreadsheets should scale the release dates axis automatically.

![number of themes and release dates](https://user-images.githubusercontent.com/7010165/42266368-7bf4faf0-7f76-11e8-84a3-c9ad566efc3a.png)
![github stars and release dates](https://user-images.githubusercontent.com/7010165/42266370-7c158d4c-7f76-11e8-950e-1b07198dd539.png)

regisphilibert commented:
This is a tight race but as of now Hugo is still number 3 “starry wise”. StaticGen is not dynamic so there is no way to really know at what their numbers will be today. Maybe Hugo will retake the lead right before they capture today’s stats but we can’t know...

Maybe rephrase to “this makes us one of the three most starred static site generator, according to...

bep commented:
>Maybe rephrase to “this makes us one of the three most starred static site generator, according to...

I have removed that reference. And added more.

regisphilibert commented:
That is one great post.

digitalcraftsman commented:
Shall we spread the word?

bep commented:
>Shall we spread the word?

I will ... thought I should wait until the americans in New york are up and running.

digitalcraftsman commented:
The eastern part of the world is / was already awake. Hugo's popularity over there should compensate that :)

bep commented:
I will Tweet now. It would be good if you could Reddit etc.?

digitalcraftsman commented:
Reddit is already done. I could submit it to Hacker News as well.
